apps_download/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=============================
+Product Download for Appstore
+=============================
+
+Providing facility for the maintain dependency of Module at level of product
+and creating Zip file with all dependent modules.
+
```

Comment:
The generating zip file feature seems in a POC state. Is it intended to base it on demo data values?

Reply:
@yvaucher This method is called from Phantom test cases and also from a Regular cron job.
only the static references in that method are for the Test case